google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -322,7 +328,13 @@ private void initEvents() {
         new HyperlinkAdapter() {
           @Override
           protected void hyperlinkActivated(HyperlinkEvent e) {
-            // TODO(ivanporty) will call ManagedSdk#update if it's installed and active.
+            CloudSdkService cloudSdkService = CloudSdkService.getInstance();
+            if (cloudSdkService instanceof ManagedCloudSdkService) {
+              ((ManagedCloudSdkService) cloudSdkService).update();
+              // do update call once and disable for visual feedback,
+              // since the following calls will essentially do nothing until update is complete.
+              checkForUpdatesHyperlink.setVisible(false);
```

Comment:
does this disable it or hide it? I'd expect it just to be disabled when clicking update.

Reply:
This is IDEA's custom component and unfortunately it completely ignores `enabled` - it's always enabled unless you hide it.

Reply:
I see. I think its a bit weird for a control to disappear after pressed. Could we consider using a different control that is disable'able?

Reply:
That's the one that IDEA uses for hyperlinks, the other one would be JXHyperlink, but it could look out of place. Let me check though. Another option is to just leave it and allow users to click it as many times as they wish - this won't do any harm to already running update but some visual feedback would be better.

Reply:
what about a button?

you point about clicking as many times as they want is good too. If its already running, you could show some sort of disappearing notice that tells them that its running already.

Reply:
it actually doesn't have to be a disappearing notice saying its running.. But if we allow them to click as many times as they want, we just need some sort of indication that the click was processed.

Reply:
sure. If you disagree that this is even an issue, we can wait until we have the bugout to decide after some hands-on time.

Reply:
I agree, I planned to just disable the hyperlink but it's really limited and hiding something on click is usually not expected.

Reply:
Updated to a button, PTAL.

Reply:
looks great